Program.cs

```diff
@@ -3,18 +3,29 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to Swap a given number
+// User gives two numbers to be swapped and the swapped numbers are displayed.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Swap numbers</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method gets the numbers from the user and displays the swapped numbers</summary>
+      static void Main () {
+         Console.WriteLine ("Enter two numbers to be swapped: ");
+         if (!double.TryParse (Console.ReadLine (), out double firstNumber)) Console.WriteLine ("Invalid Input.");

```

Comment:
If the user enters an invalid value, the program will be terminated. Instead design the program so that input can be entered until the user enters a valid input.